idd/Energy+.idd.in

```diff
@@ -5117,7 +5117,6 @@ WindowMaterial:SimpleGlazingSystem,
        \note Note that the effective upper limit for U-factor is 5.8 W/m2-K
        \units W/m2-K
        \minimum> 0
-       \maximum  7
   N2 , \field Solar Heat Gain Coefficient
        \required-field
        \note SHGC at Normal Incidence
```

Comment:
Also, where are the convection coefficient adjustment factors input additions as mentioned in the NFP? Are those being dropped for auto-calculation and not as an input anymore?

Reply:
Sorry, the NFP is not up-to-date, the adjust factor fields will not be added. I will updated the .md soon

Reply:
And a little bit extension to the question above, about two different Scenarios: 

Scenario 1: the frame U-values are indeed above 5.8 W/m2-K limit (say 6-7W/m2-K, considering some very bad thermally continuously metal frames). What would the feature help in modeling two different types of frames/windows (one is more obtrusive and one is flat)? So in this case, this feature branch will see the difference between these two cases, say if you input a U=7W/m2 for the obtrusive one, and a U=6W/m2-K for the flat; but the original develop will not see the difference, right?

Scenario 2: if the frame U-values are pretty much normal (say only about 3-4 W/m2-K). Then how the feature would deal with the difference between the two types of frames/windows (a more obtrusive frame vs. a flat frame)? In this case both the current branch and the original develop branch should model the different without any problem—say if you input a U=4W/m2-K for the obtrusive one and a U=3W/m2-K for the flat one.

Reply:
Scenario 1: yes, the users will input different U for the obtrusive and the flat one. Before this modification, the actual effective U-value reported in EnergyPlus when input U=7 is 5.838, and the reported U for U=6 is 5.834. Now, we adjusted them so that the effective U matches the input U. In addition, larger-than-seven U values could also be simulated now.

Scenario 2: when the inputs are 4 or 3, before applying the adjustment in this feature, the effective U in EnergyPlus will be 4.004 or 3.004 respectively. After the adjustment, the effective U will match the input U's (4 or 3). In this case, the adjustment ratio will be very close to 1, and the adjustment will be pretty minor.